### PR TITLE
Change how the referrer data is collected

### PR DESCRIPTION
**Pull request by WillGibson**

### CODE REVIEW

File: src/routes/handlers/send-enquiry.ts [hunk 1]
```diff
@@ -17,13 +17,15 @@ export function sendEnquiryPost(req: express.Request, res: express.Response) {
 function buildContent(enquiryId: string, req: express.Request): string {
   const firstName: string = req.body['first-name'];
   const lastName: string = req.body['last-name'];
-  const email = req.body['email'] || '';
-  const phone = req.body['phone'] || '';
-  const postcode = req.body['postcode'] || '';
```
Comment (WillGibson) [inline src/routes/handlers/send-enquiry.ts:22]: These are required fields so always present.

File: cypress/integration/SubmitAnEnquiry.feature [hunk 2]
```diff
@@ -46,15 +46,32 @@ Feature: Submit an enquiry Page
       | Phone                      |
       | How should we contact you? |
       | Postcode                   |
+      | How old are you?           |
 
-  Scenario Outline: <fieldName> is must be a valid email
-    When I fill the "<fieldName>" form element
+  Scenario: Email is must be a valid email
```
Comment (stevehenshaw) [inline cypress/integration/SubmitAnEnquiry.feature]: Typo, extra 'is' in Scenario

File: cypress/integration/SubmitAnEnquiry.feature [hunk 3]
```diff
@@ -46,15 +46,32 @@ Feature: Submit an enquiry Page
...
+    And I do not see an error message on the "referrer-email" form element
+
+  Scenario: Referrer details are required if under 18
+    When I select "under 18" radio button
+    And I click the "submit enquiry" button
+    Then I see a "First name is required" error message on the "referrer-first-name" form element
+    And I see a "Last name is required" error message on the "referrer-last-name" form element
+    And I see a "Email is required" error message on the "referrer-email" form element
+
+  Scenario: Referrer email is must be a valid email
```
Comment (stevehenshaw) [inline cypress/integration/SubmitAnEnquiry.feature]: extra 'is'

**Review by matt-lethargic — APPROVED** [review]

File: cypress/integration/common/support/textToId.js [hunk 4]
```diff
@@ -11,6 +11,8 @@ export const textToId = {
   "i'm not sure they could be anywhere": 'where-not-sure',
   'make an enquiry': 'make-an-enquiry-button',
   "no i'm not worried about searching in public": 'in-hiding-no',
+  'over 18': 'how-old-are-you',
+  'under 18': 'how-old-are-you-2',
```
Comment (stevehenshaw) [inline cypress/integration/common/support/textToId.js]: move to be alphabetical?

**Review by stevehenshaw — APPROVED** [review]
Looks cool.  Only a couple of really minor typos

File: cypress/integration/common/support/textToId.js (hunk 4 above)
Comment (WillGibson) [reply cypress/integration/common/support/textToId.js]: Done

File: cypress/integration/SubmitAnEnquiry.feature (hunk 3 above)
Comment (WillGibson) [reply cypress/integration/SubmitAnEnquiry.feature]: Done

File: cypress/integration/SubmitAnEnquiry.feature (hunk 2 above)
Comment (WillGibson) [reply cypress/integration/SubmitAnEnquiry.feature]: Done

**Pull request merged**